starlette/applications.py

```diff
@@ -47,9 +49,15 @@ def __init__(
         exception_handlers: typing.Optional[
             typing.Mapping[
                 typing.Any,
-                typing.Callable[
-                    [Request, Exception],
-                    typing.Union[Response, typing.Awaitable[Response]],
+                typing.Union[
+                    typing.Callable[
+                        [Response, Exception],
+                        typing.Union[Response, typing.Awaitable[Response]],
+                    ],
+                    typing.Callable[
+                        [WebSocket, Exception],
+                        typing.Union[None, typing.Awaitable[None]],
+                    ],
```

Comment:
Can we revert everything else in this PR and only maintain this change?

Reply:
sure.  Except, in stead of `Exception` here I think what you really want is `Any`.  Otherwise, you cannot define an exception handler taking an exception class other than Exception without type checking complaining.